website/screens/components/nav-tabs/specs/NavTabsSpecsPage.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+  DxcBulletedList,
+  DxcFlex,
+  DxcTable,
+  DxcParagraph,
+} from "@dxc-technology/halstack-react";
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Image from "@/common/Image";
+import Code from "@/common/Code";
+import navTabsSpecs from "./images/navTabs_specs.png";
+import navTabsAnatomy from "./images/navTabs_anatomy.png";
+import navTabsStates from "./images/navTabs_states.png";
+
+const sections = [
+  {
+    title: "Specifications",
+    content: (
+      <Figure caption="Nav tabs design specifications">
+        <Image src={navTabsSpecs} alt="Nav tabs design specifications" />
+      </Figure>
+    ),
+  },
+  {
+    title: "States",
+    content: (
+      <>
+        <DxcParagraph>
+          Tabs can get different states based on user interaction. States:{" "}
+          <strong>enabled</strong>, <strong>hover</strong>,{" "}
+          <strong>focus</strong>, <strong>active</strong> and{" "}
+          <strong>disabled</strong>.
+        </DxcParagraph>
+        <Image src={navTabsStates} alt="image" />
+      </>
+    ),
+  },
+  {
+    title: "Anatomy",
+    content: (
+      <>
+        <Image src={navTabsAnatomy} alt="image" />
+        <DxcBulletedList type="number">
+          <DxcBulletedList.Item>Container</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Default text label</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Selected tab indicator</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Default tab indicator</DxcBulletedList.Item>
```

Comment:
Should we refer to it as a **Default tab indicator** or it's just a simple **Divider** as in the other Tabs component?

![image](https://user-images.githubusercontent.com/44321109/224765297-b9ea0308-dc77-40e1-b70b-e43645fb4465.png)

Reply:
I think here it refers to the tab indicator, since it is not refering to the whole divider as in the tabs image.